offline-tools/copyrighter/.gitignore

```diff
@@ -1,4 +1,10 @@
+<<<<<<< HEAD
+build/
+bin/
+temp/
+=======
```

Comment:
This is a merge conflict which needs resolving.

Reply:
resolved conflicts to accept changes from origin/main